### ISSUE TITLE
JSBSim version comparison

### ISSUE THREAD

**Issue by D-ECHO** [CRJ700-family-team/CRJ700-family#8]
in this issue, I'd like to gather a list of differences and which version to prefer/use, comparing @DanielMeissner s JSBSim version (https://github.com/DanielMeissner/CRJ700-family/tree/jsbsim-dev) to mine (https://github.com/CRJ700-family-team/CRJ700-family/tree/jsb-echo)

- engine file: resolved I guess?

- Metrics (CRJ900):
https://github.com/CRJ700-family-team/CRJ700-family/blob/jsb-echo/Systems/metrics.xml and
https://github.com/CRJ700-family-team/CRJ700-family/blob/jsb-echo/Systems/metrics.xml
and
https://github.com/DanielMeissner/CRJ700-family/blob/jsbsim-dev/CRJ900jsb.xml

Organization: IMO, moving everything that is common for aircraft, into separate files to not have duplicate information is the way to go, any objections there?

Content: Daniel, you metrics have the "to be verified" comment, so would it be okay for you to use mine there, as those have been verified using the Airport Planning Manual from Bombardier.
Concerning, the AeroRP, mine is pretty sure not correct, do you know how exact yours is?
About the Eyepoint (afaik only used for pilot g force?), I think yours is pretty far back, what's the reason for this?

- Mass and Balance (CRJ900)
https://github.com/CRJ700-family-team/CRJ700-family/blob/jsb-echo/Systems/CRJ900-mass_balance.xml and https://github.com/DanielMeissner/CRJ700-family/blob/jsbsim-dev/CRJ900jsb.xml
I'd be happy to use your inertia values, as mine are pure aeromatiC++ guesses while yours seem more elaborate. 
Empty weight is same
Your CG is again pretty far backwards, is there a reason for this? Mine though isn't good, just a very very rough guess
You only have a generic payload element, I think using the elements from yasim (as in my version) would be a better start there.

Just to add, as I'm all the time writing "your" and "mine", I just don't really have a better way to express this. I don't intend to make your work seem bad or mine seem better or anything, I hope I don't create any bad mood between us there.

**Comment by DanielMeissner** [comment]
Okay I'll compare our branches in more detail soon. Just a few points to your comments:
General remark: I've only modeled the CRJ900 because that's the only one where I have sufficient data for. My plan was to make that size "perfect" and scale the other ones from there.

- Engine Files:
According to [https://en.wikipedia.org/wiki/Bombardier_CRJ700_series#Specifications] the 900 uses the 8C5. Thrust data is pure Aeromatic++ with values from the english wikipedia page for the CF34.
- Metrics:
As long as you compare the right variants, feel free to update (again, I only use -900). I measured mine from scaled 3 side view diagrams.
AeroRP: I think I used something like 1/3 chord as a base for estimation. I will check and report back.
Organization: Of course you can start separating variant files. We need to talk organisation on a larger scale as well. Hennings and my plan was to keep the CRJ900jsb in the existing CRJ700-family repo as an additional variant. We haven'd decided if we want to make the switch to full JSB FDM any time soon, so there will be a lot of variants...
- Mass Balanca:
I hope my values are ok, I used a Class II mass breakdown estimation from preliminary aircraft design (Roskam) for that and this is what we aimed for in the YASim version. Inertia values are mostly copied from there. I set the CG around that location as a first guess. I don't have a trim sheet or anything better at the moment.
- Payload
Of course, add more! Though I remember to run into an issue with compatibility to the other yasim variants and the dialog box. So I dropped that for first tests.

Naming: of course it's okay to use "mine" and "yours". This is my first JSB fdm and I'm sure there are a lot of errors and much to be learned.

**Comment by D-ECHO** [comment]
Ah, not looking properly takes revenge
I looked at https://commercialaircraft.bombardier.com/themes/bca/pdf/Bombardier_CRJ_Series_Brochure.pdf and read the engine to the 8C5B1, but that one is only used on the CRJ700, the CRJ900 indeed uses the 8C5 and the CRJ1000 uses yet another variant, the 8C5A1

**Comment by D-ECHO** [comment]
If we keep the YASim FDMs for now, I think we should put the FDMs into a separate folder, because the "main folder" is getting pretty full with the variants and it's going to be even fuller when we have (partly) two FDMs per variant

**Comment by D-ECHO** [comment]
I have now corrected/added the different engines in the jsb-echo branch

**Comment by D-ECHO** [comment]
Re: coordinates, I mostly used the ones from the YASim versions, or, e.g. for wing tips and tail structure point, got them from opening the respective 3d-model in blender. Is it possible that we are using completely different model alignments? that would explain why your coordinates are all that far backward, compared to mine

**Comment by DanielMeissner** [comment]
Yes, I used a different alignment. I recall trouble because the 3D-Origin is somewhere around the intersection of wing leading edge and fuselage. To get correct behavior and to be independent of variants (CRJ variants only add / drop fuselage elements in the front and size the planform a bit) I used that as AeroRP and made my measurements from there. 

The distances and areas are all measured should all compare to yours, actually...

**Comment by D-ECHO** [comment]
Do all of the variants share exactly the same wings or are there differences?

**Comment by DanielMeissner** [comment]
Yes, there are differences. TCDS lists for CRJ700/900/1000:
wingspan: 23,2/24,9/26,2 m
wing area: 70,6/71,1/77,4 m^2

About the empennage I don't know what's changed since I don't have measurements for other variants. The fuselage however is only stretched in the forward sections.

**Comment by D-ECHO** [comment]
hm okay. Do you think we should re-arrange the exterior 3d so that all aircraft have the AeroRP in the same place?
The tail is also partly different, the values in this branch should be accurate there as they're from the APP.
